### DISCUSSION TITLE
PANOS Proxy: show running nat-policy

### DISCUSSION THREAD

**Discussion by AmbicaY** [saltstack/salt#67621] [Feature Requests]
Need to query the NAT policy

The current execution module for panos https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.panos.html does not a get function to query the NAT policy on a PAN device. It will be really helpful to add this function. Basically an output of "show running nat-policy" command.

**Comment by krionbsd** [comment]
Hey, thank you for submitting it, as I don't have panos available to test it, could you add your code to `salt/modules/panos.py` and probably a test case into `tests/unit/modules/test_panos.py`, so we might progress on this feature request.
